lib/vagrant-cloudstack/config.rb

```diff
@@ -417,6 +422,9 @@ def finalize!
         # private ip is nil by default
         @private_ip_address     = nil if @private_ip_address == UNSET_VALUE
 
+        # expunge on destroy is nil by default
+        @expunge_on_destroy     = nil if @expunge_on_destroy == UNSET_VALUE
```

Comment:
CloudStack will ignore the extra parameters, but if the API call does not support it and a user still defines the parameter, then it will be passed to the API call and ignored. All this while the user expects the behaviour.

You are passing this to the `server.destroy` and we have to make sure this one accepts the parameter, right?